blueprint/terraform/SendSurvey_v1-0.yaml

```diff
@@ -0,0 +1,106 @@
+surveyInvite:
```

Comment:
Generally, only the markdown file and images that go with it go in the `blueprint` directory. The technical materials that are the actual "blueprint" go in the root of the repo in whatever structure is best for the content. Everything in the `blueprint` directory gets copied into the dev center's content structure, so don't include these files in this directory unless you intend for them to be linked to in the blueprint article (which you aren't doing currently as far as I can tell).